lib/compat/wordpress-6.9/terms-query-block-patterns.php

```diff
@@ -0,0 +1,101 @@
+					<div class="wp-block-terms-query">
+					<!-- wp:term-template {"style":{"spacing":{"blockGap":"var:preset|spacing|40"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch","flexWrap":"nowrap"}} -->
+					<!-- wp:group {"layout":{"type":"flex","orientation":"vertical","flexWrap":"nowrap","justifyContent":"stretch"}} -->
+					<div class="wp-block-group">
+					<!-- wp:heading {"metadata":{"bindings":{"content":{"source":"core/term-data","args":{"key":"name"}}}}} -->
+					<h2 class="wp-block-heading"></h2>
+					<!-- /wp:heading -->
+					<!-- wp:paragraph {"metadata":{"bindings":{"content":{"source":"core/term-data","args":{"key":"description"}}}}} -->
+					<p></p>
+					<!-- /wp:paragraph -->
+					<!-- wp:buttons -->
+					<div class="wp-block-buttons">
+					<!-- wp:button {"metadata":{"bindings":{"url":{"source":"core/term-data","args":{"key":"link"}}}},"className":"is-style-fill"} -->
+					<div class="wp-block-button is-style-fill"><a class="wp-block-button__link wp-element-button">View posts</a></div>
+					<!-- /wp:button -->
+					</div>
+					<!-- /wp:buttons -->
+					</div>
+					<!-- /wp:group -->
+					<!-- /wp:term-template -->
+					</div>
+					<!-- /wp:terms-query -->',
+			),
+			'term-query-cat-list'         => array(
+				'title'      => _x( 'Category List', 'Block pattern title' ),
+				'blockTypes' => array( 'core/terms-query' ),
+				'categories' => array( 'query' ),
+				'content'    => '<!-- wp:terms-query {"termQuery":{"taxonomy":"category","hierarchical":true}} -->
+					<div class="wp-block-terms-query">
+					<!-- wp:term-template {"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch","flexWrap":"nowrap"}} -->
+					<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"left","verticalAlignment":"center"}} -->
+					<div class="wp-block-group">
+					<!-- wp:buttons {"layout":{"type":"flex","verticalAlignment":"center"}} -->
+					<div class="wp-block-buttons">
+					<!-- wp:button {"metadata":{"bindings":{"url":{"source":"core/term-data","args":{"key":"link"}},"text":{"source":"core/term-data","args":{"key":"name"}}}},"className":"is-style-outline","style":{"spacing":{"padding":{"left":"0","right":"0","top":"0","bottom":"0"}},"border":{"color":"#00000000","radius":{"topLeft":"0px","topRight":"0px","bottomLeft":"0px","bottomRight":"0px"}}}} -->
+					<div class="wp-block-button is-style-outline"><a class="wp-block-button__link has-border-color wp-element-button" style="border-color:#00000000;border-top-left-radius:0px;border-top-right-radius:0px;border-bottom-left-radius:0px;border-bottom-right-radius:0px;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0">View posts</a></div>
+					<!-- /wp:button -->
+					</div>
+					<!-- /wp:buttons -->
+					<!-- wp:paragraph {"metadata":{"bindings":{"content":{"source":"core/term-data","args":{"key":"count"}}}},"fontSize":"medium"} -->
```

Comment:
This `fontSize` is being inserted as `large` when I use this pattern. I'm not sure why, as the theme I'm testing with does have a `medium` font size preset:

<img width="1495" height="428" alt="image" src="https://github.com/user-attachments/assets/60431628-72a9-4f2c-8e8a-f7c0d9acf24a" />